src/Message/VirusScan.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Message;
+
+/**
+ * Message for virus-scanning files.
+ */
+class VirusScan
```

Comment:
So the message are of type "command" and should be names a command. Or else it will be very confusing. Something like: 'ScanFileForVirus' or 'doVirusScanOnFile' or 'VirusScanFile' even maybe?